### PR TITLE
Workshop rover

### PR DESCRIPTION
**Pull request by loopDelicious**
This workshop follows Drive a Rover w/TS tutorial, but we can add expansion pack(s) for CV et al in the future

- cherry picked a commit from another PR that's not ready yet (my most triumphant git success to date), but blocking a test to release into production
- please also review the part on wifi hotspot for take-home projects

### CODE REVIEW

**Review by HipsterBrown — CHANGES_REQUESTED** [review]
This is in really good shape! Just a few suggested fixes for Pi 5 vs Pi 4 and accidental inclusion of "air sensor".  

---

I wonder how much we could lean into fragments for other workshop codelabs as well. We still want to teach how to configure machines, but for more complex / extensive configurations like the rover, they can be a good way to streamline that process. Or show the teachers how to create them for their workshops.

**Review by HipsterBrown — CHANGES_REQUESTED** [review]
This is in really good shape! Just a few suggested fixes for Pi 5 vs Pi 4 and accidental inclusion of "air sensor".  

---

I wonder how much we could lean into fragments for other workshop codelabs as well. We still want to teach how to configure machines, but for more complex / extensive configurations like the rover, they can be a good way to streamline that process. Or show the teachers how to create them for their workshops.

File: site/viamguides/src/workshop-rover/workshop-rover.md [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+Duration: 5
+
+Review the list of required components (hardware, software), and determine which materials are provided or required for participants to bring on their own. Provide instructions to participants, giving them ample time to prepare and complete prerequisites.
+
+### Hardware and supplies requirements
+
+- 1 - [Viam rover](https://docs.viam.com/appendix/try-viam/rover-resources/)
+  > aside negative
+  > You can also use [another wheeled robot like Yahboom or SCUTTLE](https://docs.viam.com/tutorials/configure/configure-rover/), or go to [Try Viam](https://app.viam.com/try) in your web browser to remotely control a rover pre-configured with components in the NYC Viam robotics lab.
+- 1 - [Raspberry Pi 4](https://a.co/d/fc3JUMm)
```
[the remark this answers is not in the document]
Comment (loopDelicious) [reply site/viamguides/src/workshop-rover/workshop-rover.md:52]: I thought you said market share and accessibility / affordability was higher for Pi4? 

But yes, all the docs indicate the Viam rover was designed for Pi4, and "support for other boards is available but additional components are needed"

File: site/viamguides/src/workshop-rover/workshop-rover.md [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+### Set up your Raspberry Pi
+
+The Raspberry Pi boots from a USB flash drive (or microSD card). You need to install Raspberry Pi OS on a USB flash drive that you will use with your Pi. For more details about alternative methods of setting up your Raspberry Pi, refer to the [Viam docs](https://docs.viam.com/installation/prepare/rpi-setup/#install-raspberry-pi-os).
+
+### Install Raspberry Pi OS
+
+1. Connect the USB flash drive (or microSD card) to your computer.
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) and launch it.
+   ![raspberry pi imager](assets/imager.png)
+1. Click **CHOOSE DEVICE**. Select your model of Pi, which is Raspberry Pi 5.
```
[the remark this answers is not in the document]
Comment (loopDelicious) [reply site/viamguides/src/workshop-rover/workshop-rover.md:148]: This was an oversight, I was working with a 4.

**Pull request merged**